app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
```

Comment:
probably no needed use brackets here if don't wtire each import in individual line

Reply:
fix imports according to pep8
https://peps.python.org/pep-0008/#imports:~:text=Imports%20should%20be,group%20of%20imports.